gcloud-java-examples/README.md

```diff
@@ -67,6 +67,16 @@ To run examples from your command line:
   $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.ResourceManagerExample" -Dexec.args="get my-project-id"
   ```
 
+  Here's an example run of `BigQueryExample`.
+
+  Before running the example, go to the [Google Developers Console][developers-console] to ensure that Google Cloud BigQuery API is enabled.
```

Comment:
Nit: I think the name of the service to enable in the Google Developers Console is "BigQuery API" (without the "Google Cloud" prefix).